keygen/Cargo.toml

```diff
@@ -20,3 +20,6 @@ solana-sdk = { path = "../sdk", version = "0.13.0" }
 name = "solana-keygen"
 path = "src/keygen.rs"
 
+[[bin]]
+name = "solana-pubkeygen"
```

Comment:
So how about we stick with a single `solana-keygen` and it works on both keypairs and pubkeys (hey there's a "key" substring in each right!)

```bash
$ solana-keygen            # same behaviour as today (really a shortcut for `solana-keygen new`)
$ solana-keygen new        # same as `solana-keygen`
$ solana-keygen new -o -   # same as `solana-keygen -o -`

$ solana-keygen print                                # prints the pubkey from ~/.config/solana/id.json
$ solana-keygen print my_keypair.json                # prints pubkey from my_keypair.json
$ solana-keygen print my_keypair.json -o pubkey.json # reads pubkey from my_keypair.json and saves it to pubkey.json
```